modules/pel/peltool/peltool.py

```diff
@@ -335,7 +335,18 @@ def listOption(path: str, config: Config, extension: str, rev: bool =False):
             file_list.append(file)  ## create list of file names
         # Only process top level directory
         break
-    file_list.sort(reverse=rev) 
+    file_list.sort(reverse=rev)
+    if all:
```

Comment:
I feel we are complicating the code...

Please define a function for each option and use it.

If there is common code between options please define some helper function use it.